rest/tests/test_rest.py

```diff
@@ -321,24 +315,25 @@ def test_term(self):
         # never created a duplicate
         self.assertEqual(1, Term.objects.count())
 
-        # test missing vocabulary
-        # term permission work not done
-        self.get_term(self.repo.slug, vocab.slug, "missing",
+        # test missing repository
+        self.get_term(self.repo.slug, vocab_slug, "missing",
                       expected_status=HTTP_404_NOT_FOUND,
                       skip_options_head_test=True)
-        self.assertEqual(0, self.get_terms(self.repo.slug, "missing")['count'])
-
-        vocab2 = create_vocabulary(
-            self.repo.id,
-            "other name",
-            "description",
-            True,
-            Vocabulary.FREE_TAGGING,
-            1000
-        )
-
-        self.delete_term(self.repo.slug, vocab.slug, new_term_slug,
-                         expected_status=HTTP_405_METHOD_NOT_ALLOWED)
+        self.get_terms(self.repo.slug, "missing",
+                       expected_status=HTTP_404_NOT_FOUND)
+
+        vocab2_slug = self.create_vocabulary(
+            self.repo.slug,
+            {
+                "name": "other name",
+                "description": "description",
+                "required": True,
+                "vocabulary_type": Vocabulary.FREE_TAGGING,
+                "weight": 1000,
+            }
+        )['slug']
+
+        self.delete_term(self.repo.slug, vocab_slug, new_term_slug)
```

Comment:
Should there be a check here that the term is actually gone, such as deleting it again (or getting it again) and receiving `NOT_FOUND`?